test/framework/filetools.py

```diff
@@ -2027,19 +2027,35 @@ def makedirs_in_test(*paths):
 
     def test_find_eb_script(self):
         """Test find_eb_script function."""
+
+        # make sure $EB_SCRIPT_PATH is not set already (used as fallback mechanism in find_eb_script)
+        if 'EB_SCRIPT_PATH' in os.environ:
```

Comment:
Don't you need to reset this to the original value at the end of this test?

Reply:
The environment is already automatically reset to the original at the end of each test (see `tearDown` in `test/framework/utilities.py`).